src/components/js-tracer/Canvas.js

```diff
@@ -42,7 +43,9 @@ type OwnProps = {|
   +rowHeight: CssPixels,
   +threadIndex: ThreadIndex,
   +doFadeIn: boolean,
-  +updatePreviewSelection: typeof updatePreviewSelection,
+  +updatePreviewSelection: WrapFunctionInDispatch<
+    typeof updatePreviewSelection
+  >,
```

Comment:
This is an example of a wrapped ThunkAction being passed into another component through the `OwnProps`.